tests/test_spark.py

```diff
@@ -389,3 +389,64 @@ def test_task_stdout():
         assert stdout_file["size"] > 0, "stdout file should have content"
     finally:
         sdk_install.uninstall(utils.SPARK_PACKAGE_NAME, service_name)
+
+
+@pytest.mark.sanity
+def test_supervise_conflict_frameworkid():
+    job_service_name = "MockTaskRunner"
+
+    @retrying.retry(
+        wait_fixed=1000,
+        stop_max_delay=600 * 1000,
+        retry_on_result=lambda res: not res)
+    def wait_job_present(present):
+        svc = shakedown.get_service(job_service_name)
+        if present:
+            return svc is not None
+        else:
+            return svc is None
+
+    job_args = ["--supervise",
+                "--class", "MockTaskRunner",
+                "--conf", "spark.cores.max=1",
+                "--conf", "spark.executors.cores=1"]
+
+    try:
+        driver_id = utils.submit_job(app_url=utils.dcos_test_jar_url(),
+                app_args="1 1800",
+                service_name=utils.SPARK_SERVICE_NAME,
+                args=job_args)
+        log.info("Started supervised driver {}".format(driver_id))
+
+        wait_job_present(True)
+        log.info("Job has registered")
+
+        sdk_tasks.check_running(job_service_name, 1)
+        log.info("Job has running executors")
+
+        service_info = shakedown.get_service(job_service_name).dict()
+        driver_regex = "spark.mesos.driver.frameworkId={}".format(service_info['id'])
+        status, stdout = shakedown.run_command_on_agent(service_info['hostname'], 
+                "ps axo pid,command | grep -v grep | grep '{}' | cut -d' ' -f1 | sudo xargs kill -9".format(driver_regex),
+                username=sdk_cmd.LINUX_USER)
+
+        if status:
+            log.info("Killed all the PIDs")
+        else:
+            log.info("Unable to kill PIDs")
```

Comment:
let's remove this conditional logging after switching to `sdk_cmd.kill_task_with_pattern`

Reply:
Removed it.